### DISCUSSION TITLE
Networking Support

### DISCUSSION THREAD

**Discussion by ianatha** [ianatha/bababasic#78] [General & Ideas]
It'd be awesome to have some simple TCP/IP support while staying true to the spirit of the language.

Just off the top of my head something like the following feels tractable:

```
OPEN "TCP:\127.0.0.1\80\LISTEN" FOR INPUT AS #1
INPUT #1, CONN$
IF CONN$ <> ""
  REM CONN$ contains something like "TCP:\127.0.0.1\80\REMOTE_HOST\REMOTE_PORT"
  OPEN CONN$ AS #2

  REM ... respond to request
  PRINT #2, "200 OK"
  CLOSE #2
END IF
```

**Comment by stuaxo** [comment]
Does QB64 support networking by default ?

It would be nice to try and be compatible where possible - I remember using BASIC as a kid and it was a pain trying another BASIC and finding stuff didn't work.
